esi_leap/common/idp.py

```diff
@@ -23,25 +22,26 @@
 _cached_project_list = None
 
 
-def get_keystone_client():
+def get_idp_client():
     global _cached_keystone_client
     if _cached_keystone_client is not None:
         return _cached_keystone_client
 
-    auth_plugin = ks_loading.load_auth_from_conf_options(CONF, "keystone")
-    sess = ks_loading.load_session_from_conf_options(CONF, "keystone", auth=auth_plugin)
-    cli = keystone_client.Client(session=sess)
+    # Get Client config option
+    module_path, class_name = CONF.esi.idp_plugin_class.rsplit(".", 1)
+    module = importlib.import_module(module_path)
+    cli = getattr(module, class_name)()
```

Comment:
This feels a little delicate - can you do something similar to what we do for resource objects? (https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/objects/lease.py#L313 which calls https://github.com/CCI-MOC/esi-leap/blob/master/esi_leap/resource_objects/__init__.py#L34)